README.md

```diff
@@ -302,8 +302,12 @@ The following hooks are currently available:
 
 Every hook will have the name of the container for which this hook runs available as the environment variable `CRANE_HOOKED_CONTAINER`.
 
+### Global Flags
```

Comment:
I think I actually prefer the old structure. Can you remove this headline and promote the 2 headlines below?

I would remove the paragraph directly below.